src/site/stages/build/process-cms-exports/transformers/node-press_release.js

```diff
@@ -0,0 +1,74 @@
+const moment = require('moment');
+
+const {
+  createMetaTagArray,
+  getDrupalValue,
+  getWysiwygString,
+  isPublished,
+} = require('./helpers');
+
+const transform = entity => ({
+  entityType: 'node',
+  entityBundle: 'press_release',
+  title: getDrupalValue(entity.title),
+  entityMetatags: createMetaTagArray(entity.metatag.value),
+  entityUrl: {
+    path: entity.path[0].alias,
+  },
+  fieldAddress: {
+    locality: entity.fieldAddress[0].locality,
+    // The things in fieldAddress don't get camelCased
+    administrativeArea: entity.fieldAddress[0].administrative_area,
+  },
+
+  fieldIntroText: getDrupalValue(entity.fieldIntroText),
+  fieldOffice: entity.fieldOffice[0],
+  fieldPdfVersion: entity.fieldPdfVersion[0] || null,
+  fieldPressReleaseContact: entity.fieldPressReleaseContact[0]
+    ? [
+        {
+          entity: {
+            title: `${entity.fieldPressReleaseContact[0].fieldNameFirst} ${
+              entity.fieldPressReleaseContact[0].fieldLastName
+            }`,
+            fieldDescription:
+              entity.fieldPressReleaseContact[0].fieldDescription,
+            fieldPhoneNumber:
+              entity.fieldPressReleaseContact[0].fieldPhoneNumber,
+            fieldEmailAddress:
+              entity.fieldPressReleaseContact[0].fieldEmailAddress,
+          },
+        },
+      ]
+    : [],
```

Comment:
Should I modify the `node-person_profile` to include a `title` and just make this whole thing `entity.fieldPressReleaseContact[0] || []` with the schema being an array with `node-person_profile` items?

Reply:
That _sounds_ like a better solution since we won't have entities that may look different depending on their parents (like it is now :roll_eyes:).

I reserve the right to regret saying this, though. :sweat_smile: